crt_portal/cts_forms/views.py

```diff
@@ -339,6 +350,26 @@ def serialize_data(report, request, report_id):
     return output
 
 
+class ProfileView(LoginRequiredMixin, FormView):
+    """Can be used for updating section filter for a profile"""
+    form_class = ProfileForm
+
+    def post(self, request):
+        """Update or create Profile"""
+        if hasattr(request.user, 'profile'):
+            instance = request.user.profile
+        else:
+            instance = Profile()
+            instance.user = request.user
+
+        profile_form = ProfileForm(request.POST, instance=instance)
+        if profile_form.is_valid() and profile_form.has_changed():
+            """Save Data in database"""
+            profile_form.save()
+        """redirects back to /form/view but all filter params are not perserved. """
```

Comment:
Very minor thing, but generally triple quotes are for long comments (with newlines and/or tabs), please use `#` instead.